src/main/java/com/aca/homework/week15/sql/generate/SqlGenerator.java

```diff
@@ -0,0 +1,33 @@
+package com.aca.homework.week15.sql.generate;
+
+import java.lang.reflect.Field;
+
+public class SqlGenerator {
+    public String generateInsert(User user) {
```

Comment:
When a new field is added in the User class this stops working. Please write a more general code that using the reflection will build SQL for all declared fields